addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js

```diff
@@ -25,8 +25,10 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
 
     isProjectTasksContext() {
         return (
-            ["project.project", "project.task.type.delete.wizard"].includes(
+            (["project.project", "project.task.type.delete.wizard"].includes(
                 this.props.list.context.active_model
+            )
+            || !!this.props.list.context.active_id
```

Comment:
it is not possible to know the active_model to be sure we come from the right model?
Maybe in the search model?

Reply:
Thanks for the review. I m not sure if I totally get how can we get the model we come from when we copy past the link. I was thinking about making getting the action external Id and allowing the quick button to appear if it matches the one used when we come from a project (in this case 'project.act_project_project_2_project_task_all'). WDYT?

Reply:
I don't think it is a good idea since we could come from another action and have the same behavior, we have to be more robust.

Reply:
Do you think that would be a problem? Since even if we came from another action the behavior would still be correct. Quick add button is added when we are in kanban view filtered on project and grouped by stage.